src/main/resources/stubs/strconv/atoi.gobra

```diff
@@ -25,12 +25,14 @@ type NumError struct {
 requires p > 0
 requires acc(e, p)
 ensures acc(e, p)
+decreases _
 func (e *NumError) Error(ghost p perm) string /*{
 	return "strconv." + e.Func + ": " + "parsing " + Quote(e.Num) + ": " + e.Err.Error()
 }*/
```

Comment:
right now, it is not sound to assume that this terminates becaure Gobra does not guarantee that the Error method in `error` does not terminate

Reply:
ok, in that case I will remove this contract